lib/apiservers/portlayer/swagger.yml

```diff
@@ -249,6 +249,8 @@ paths:
             $ref: "#/definitions/VolumeResponse"
         '500':
           description: "Error"
+          $schema:
+            $ref: "#/definitions/Error"
         default:
           description: "error"
```

Comment:
The status code is 500 @sflxn.  And as long as the response is relevant, I don't see the code as being a problem unless you want to map all possible error codes (http://www.gnu.org/software/libc/manual/html_node/Error-Codes.html) to an http error code.

In any case, we're not going to address this here.  Please create an issue and describe an alternative.